wordpress/wp-content/plugins/memberful-wp/src/paywall/renderer.php

```
@@ -0,0 +1,346 @@
+	 *
+	 * @return string
+	 */
+	private static function wrapper_style( array $config ): string {
+		$parts = array( '--mf-radius:' . self::button_radius( $config['button_shape'] ) );
+
+		if ( ! empty( $config['brand_color'] ) ) {
+			$parts[] = '--mf-brand:' . $config['brand_color'];
+		}
+
+		return implode( ';', $parts ) . ';';
+	}
```

PHPMD (2.15.0)

[error] The method wrapper_style is not named in camelCase. (undefined)

(CamelCaseMethodName)